MachObfuscator/Options/Options.swift

```diff
@@ -178,7 +178,7 @@ extension Options {
           -d, --debug             debug mode, output more verbose info to stdout
           --dry-run               analyze only, do not save obfuscated files
         
-          -t, --methtype          obfuscate methType section (objc/runtime.h methods may work incorrectly)
+          -t, --methtype          erase methType section (objc/runtime.h methods may work incorrectly)
```

Comment:
MachObfuscator has no stable release so far. What do you think about breaking the interface compatibility by changing this option to `--erase-methtype`? We could be extra nice by providing a warning that `--methtype` option has been replaced, but I don't think this is really needed.

Reply:
I may change option name and remove `-t` as well.

Old option will stop working with clear message so if someone uses it, they will see help text. I think this should be sufficient.

Reply:
done